urlarbiter/url_arbiter.go

```diff
@@ -57,14 +58,17 @@ func (u *URLArbiter) Register(path, publishingAppName string) (URLArbiterRespons
 		return URLArbiterResponse{}, err
 	}
 
-	// Read the response body and then check the status code so we can
-	// return the errors from the response.
-	switch response.StatusCode {
-	case 422: // Unprocessable Entity.
-		return arbiterResponse, UnprocessableEntity
-	case http.StatusConflict:
-		return arbiterResponse, ConflictPathAlreadyReserved
+	if response.StatusCode < 300 && response.StatusCode >= 200 {
+		// 1xx and 2xx
```

Comment:
this comment doesn't match the condition; best to remove it, the conditional is easily readable. one more minor pedantic change, it'd be better to have the lower-bound (`>= 200`) condition first and upper-bound (`< 300`) next.